libraries/ui/src/Nav.tsx

```diff
@@ -14,92 +14,152 @@ export type NavProps = React.PropsWithChildren<{
+    <path d="M27.1421 28.1421L30.6776 31.6776" stroke="currentColor" strokeWidth="2" strokeLinecap="round" />
+  </svg>
+);
+
+const DropdownIcon: React.FC<{ expanded: boolean }> = ({ expanded }) => (
+  <svg
+    className={clsx('size-4 transition-transform', expanded ? 'rotate-180' : '')}
+    viewBox="0 0 24 24"
+    fill="none"
+    stroke="currentColor"
+  >
+    <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 9l-7 7-7-7" />
+  </svg>
+);
+
+const ExploreSection: React.FC<{
+  expanded: boolean;
+  innerClassName?: string;
+  className?: string;
+  courses: Array<{ title: string; href: string; isNew?: boolean }>;
+}> = ({
+  expanded, innerClassName, className, courses,
+}) => (
+  <div className={clsx(
+    'nav-explore-section__content-wrapper overflow-hidden transition-[max-height,opacity] duration-300',
+    expanded ? 'max-h-96 opacity-100' : 'max-h-0 opacity-0',
+    className,
+  )}
+  >
+    <div className={clsx('nav-explore-section___dropdown-content flex flex-col gap-[14px] w-fit overflow-hidden text-pretty', innerClassName)}>
+      <h2 className="nav-explore-section__dropdown-title font-bold text-xl">Our courses</h2>
+      {courses?.map((course) => (
+        <a key={course.href} href={course.href} className="nav-explore-section__dropdown-link">
+          {course.isNew && (
+          <span className="nav-explore-section__new-badge text-bluedot-normal text-xl font-black pr-2">
+            New!
+          </span>
+          )}
+          {course.title}
+        </a>
+      ))}
+    </div>
+  </div>
+);
+
+const NavLinks: React.FC<{
+  exploreSectionInline?: boolean,
+  children: React.ReactNode,
+  courses: Array<{ title: string; href: string; isNew?: boolean }>,
+  exploreExpanded: boolean,
+  onToggleExplore: () => void,
+  className?: string
+}> = ({
+  exploreSectionInline, children, courses, exploreExpanded, onToggleExplore, className,
+}) => (
+  <div className={clsx('nav-links flex gap-9', className)}>
+    <div>
+      <button
+        type="button"
+        onClick={onToggleExplore}
+        className="nav-links__dropdown-button flex items-center gap-2 hover:text-bluedot-normal"
+      >
+        Explore
+        <DropdownIcon expanded={exploreExpanded} />
+      </button>
+      {exploreSectionInline && <ExploreSection expanded={exploreExpanded} courses={courses} className="nav-links__explore-section" innerClassName="pl-6 pt-6" />}
+    </div>
+    {children}
+  </div>
+);
+
+const LoginButtons: React.FC<{ className?: string }> = ({ className }) => (
+  <div className={clsx('nav__cta-container flex items-center', className)}>
+    <a className="nav__secondary-cta" href="https://course.aisafetyfundamentals.com/alignment?show=login" {...EXTERNAL_LINK_PROPS}>Log in</a>
+    {/* TODO flag decision to keep default here */}
+    <CTALinkOrButton className="nav__primary-cta" url="https://aisafetyfundamentals.com/" {...EXTERNAL_LINK_PROPS}>Get started for free</CTALinkOrButton>
```

Comment:
I decided to stick with the primary button, rather than adding the tertiary one as shown in the designs:
![Screenshot 2025-02-06 at 23 39 23](https://github.com/user-attachments/assets/c3d47db1-3fde-43e3-b3de-0eef16808533)
^In the designs, this one has font-weight: 500, and font-size: 16px, which deviates from the standard (and is fiddly to override with tailwind classes). Without these adjustments it doesn't look very good imo, whereas the primary button looks good with the default styling